src/mscorlib/src/System/Threading/ExecutionContext.cs

```diff
@@ -694,7 +685,7 @@ enum Flags
         }
         private Flags _flags;
 
-        private Dictionary<IAsyncLocal, object> _localValues;
+        private IAsyncLocalValueMap _localValues;
```

Comment:
Ok.  Just to be clear, you mean removing all code that's specific to !FEATURE_CORECLR, right?  Separately, are we planning to do that systematically across CoreLib?